curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8d539dc11cb42b5dd7ec8.md

```diff
@@ -7,7 +7,14 @@ dashedName: step-104
 
 # --description--
 
-The `shift()` method on an array removes the first element in the array and returns it. Use this method to take the first element from the `inventory` array and assign it to your `currentWeapon` variable.
+The `shift()` method on an array removes the first element in the array and returns it.
+
+```js
+const numbers = [1, 2, 3];
+const firstNumber = numbers.shift(); // returns 1
```

Comment:
Hmm, I'm not sure about comment here. It might be better to add another sentence with something like:

> After executing code below/above, the `firstNumber` will be equal to `1`.

@jdwilkin4 what do you think? This also would limit issues with translating, altought there isn't really that much to translate in `returns 1`.

Reply:
I think either option works. 
We can either have another sentence or leave the comment.
We have had comments in other projects like this one 

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-string-and-array-methods-by-building-a-music-player/step-9

Reply:
Ah, then this is fine with me.